R-package/R/lgb.cv.R

```diff
@@ -9,10 +9,11 @@ CVBooster <- R6::R6Class(
     boosters = list(),
     initialize = function(x) {
       self$boosters <- x
+      return(invisible(NULL))
     },
     reset_parameter = function(new_params) {
       for (x in boosters) { x$reset_parameter(new_params) }
-      self
+      return(self)
```

Comment:
```suggestion
      return(invisible(self))
```

Since this class doesn't have a print method implemented, I have a slight preference for `return(invisible(self))`.